pandas/tests/io/json/test_compression.py

```diff
@@ -21,6 +24,44 @@ def test_compression_roundtrip(compression_no_zip):
         assert_frame_equal(df, pd.read_json(result))
 
 
+@pytest.mark.parametrize('filename,compression,compressor', [
+    ("test.gz", "gzip", gzip.GzipFile),
+    ("test.gz", "infer", gzip.GzipFile),
+    ("test.bz2", "bz2", bz2.BZ2File),
+    ("test.bz2", "infer", bz2.BZ2File)
+])
+def test_to_json_compression(filename, compression, compressor):
+    # see GH issue 15008
+
+    df = pd.DataFrame({"A": [1]})
+    exp = b'{"A":{"0":1}}'
+
+    with tm.ensure_clean(filename) as path:
+        df.to_json(path, compression=compression)
+        with compressor(path, "rb") as f:
+            assert f.read() == exp
+
+        tm.assert_frame_equal(pd.read_json(path, compression="infer"), df)
+
+
+@td.skip_if_no_lzma
```

Comment:
compression is now a top level fixture so this needs to be updated

Reply:
Yep, done.